docs/debugging-tips.rst

```diff
@@ -0,0 +1,56 @@
+.. _Debugging Tips:
+
+**************
+Debugging Tips
+**************
+
+.. _CloudWatch Logs Filters:
+
+CloudWatch Logs Filters
+***********************
+
+KMS CMK Usage
+=============
+
+The application provides some filters to find events for your KMS CMK ARN. Here are some additional filters to explore
+KMS calls in the application to understand how the key is being used.
+
+Note that these filters will show all matching events, including for any other KMS keys in your account that are not
+being used in this workshop. If you want to scope it down to just the workshop's ARN,
+add an additional constraint, such as ``&& $.resources[0].ARN = "arn:aws:kms:us-east-2:<account>:key/<key>"``.
```

Comment:
Maybe add a note to replace the placeholder values with real values?

Reply:
Good idea, added